getting-started/installation-windows-env.rst

```diff
@@ -44,11 +44,11 @@ If you have Administrator rights and want to do it globally, you can also select
 
 1. Select ``Advance`` tab, navigate to ``Environment Variables`` button. That should open editable tables of settings for Windows Environment. 
 2. Select ``User Variable for 'user'`` option, press NEW
-3. Add EPICS BASE path here. In ``Variable Name``, put "EPICS_BASE". For ``Variable Value``, enter the location of your EPICS Base installation, e.g.,  "C:\\msys64\\home\\'user'\\base-R7.0.4.1"
+3. Add EPICS BASE path here. In ``Variable Name``, put "EPICS_BASE". For ``Variable Value``, enter the location of your EPICS Base installation, e.g.,  "c:\\msys64\\home\\ 'user'\\base-R7.0.4.1"
```

Comment:
I know that the drive letter is accepted in lower case, but the "official" notation (as used in system environment variables etc.) is upper case.

A space character in the path before the user name seems wrong.

Reply:
Ok. It shall remain Capital.

Problem with this path is that when published on 'readthedocs'  '\' between "home' and 'user' is not visible. It shows ok in github preview though. Only if I put a space path is shown correct. But then as you said it shows space in the path.  Can you try something to make it look correct ?

Reply:
Oh, ok.
I will check how this could be done.